api-reference/10 UI Components/dxPopover/1 Configuration/hideOnParentScroll.md

```diff
@@ -5,7 +5,9 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+<!-- %shortDescription% -->
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+<!-- Description goes here -->
+
+<!-- import * from 'api-reference\10 UI Components\dxOverlay\1 Configuration\hideOnParentScroll.md' -->
```

Comment:
Popover inherits Overlay via Popup. You can safely remove the short and full decriptions - they will be inherited.